week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -12,12 +12,15 @@ let bottle = {
   volume: 0,
   fill: function() {
     // calling this function should make you bottles volume = 100;
+    this.volume = 100;
   },
   drink: function() {
     // calling this function should decrease your bottles volume by 10;
+    this.volume -= 10;
   },
   empty: function() {
     // this function should return true if your bottles volume = 0
+    this.volume === 0 ? true : false;

```

Comment:
use of ternary operate 👍

Reply:
@dahfool  i certainly disagree here, as the Equality operator always return Boolean value so there's not need to explicit `true` or `false` in first place